crates/reporting/tests/test_reporting.rs

```diff
@@ -4586,6 +4586,28 @@ mod test_reporting {
     "###
     );
 
+    test_report!(
+        expression_indentation_end,
+        indoc!(
+            r#"
+            f <- Foo.foo
+            "#
+        ),
+        @r###"
+    ── INDENT ENDS AFTER EXPRESSION ────── tmp/expression_indentation_end/Test.roc ─
+
+    I am partway through parsing an expression, but I got stuck here:
+
+    1│  app "test" provides [main] to "./platform"
+    2│
+    3│  main =
+    4│      f <- Foo.foo
+                        ^
+
+    Looks like the indentation ends prematurely after an expression
```

Comment:
Can we add a period and a suggestion for what we'd expect here, e.g. something like 
```suggestion
    Looks like the indentation ends prematurely here. Did you mean
    to have another expression after this line?
```
?